docker-compose.yml

```diff
@@ -2,7 +2,7 @@ version: "3.9"
 name: tiros
 services:
   ipfs:
-    image: ipfs/kubo:v0.19.0
+    image: ${IPFS_IMAGE:-ipfs/kubo:v0.19.0}
```

Comment:
allows overriding the image easily, so I can rebuild helia-http-gateway locally and have it update here.